packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -100,8 +105,8 @@ const useStyles = makeStyles(({ palette, spacing, typography }) => ({
   iconContainer: {
     marginLeft: spacing(2)
   },
-  icon: { height: 20, width: 20 },
-  removeIcon: { height: 24, width: 24 },
+  icon: { height: spacing(6), width: spacing(6) },
+  removeIcon: { height: spacing(6), width: spacing(6) },
```

Comment:
the values for icon height and width changed, making sure thats intentional. also seems weird to use spacing for the height and width of things 🤔

Reply:
yes it was intentional, and good point about using spacing, i just went with it to be consistent with other times ive seen it used for height and width....

Reply:
ewww, sounds good

Reply:
lolll ill change it, please please sir i just found it on the ground